docs/rewards/consensus-fee.md

```diff
@@ -0,0 +1,40 @@
+---
+sidebar_position: 4
+---
+
+# Minimum Consensus Fee Overview  
+
+The minimum consensus fee is an important part of transacting on the Archway Network. This guide will explain what the minimum consensus fee is and how to get this information.
+
+## Minimum Consensus Fee Definition 
+
+The minimum consensus fee is the lowest required amount a user pays in transaction fees. Transactions with a lower fee amount than the minimum fee are declined. 
+
+This minimum fee protects the economic model of Archway. It prevents the incentive of sending low/no fee transactions to gain higher dapp rewards. 
+
+This minimum fee is shown in one gas unit, for example, `0.002 uarch`. The client should query the fee before submitting a new transaction. 
+
+## Minimum Fee Calculation 
+
+The minimum consensus fee is calculated for each new block and shown in one gas unit like `0.002` arch. The formula for the calculating the fee is below; 
+
+ `Inflation Block Rewards` /  `Block Gas Limit` * `Transaction Fee Rebate Ratio` - `Block Gas Limit` 
+
+ `Inflation Block Rewards` - Inflationary rewards per block.  <br /> 
+ `Block Gas Limit` - Maximum gas limit per block. <br /> 
+ `Transaction Fee Rebate Ratio`- Ratio of split fees between Validators and dapps.
+
+
+
+  ## Querying the Minimum Consensus Fee 
+
+Use the below query to get the estimated transaction fee for a given transaction. This will return the minimum consensus fee for the current block: 
+
+``bash 
+ archwayd query rewards estimate-fees [gas-limit] [flags]
+``
+
+### Example Query 
+``bash
+archwayd q rewards estimate-fees 1 --node 'https://rpc.titus-1.archway.tech:443' --output json | jq -r '.gas_unit_price | (.amount + .denom)'
```

Comment:
when running this command,  i am getting
` Error: unknown command "rewards" for "q"` or (`Error: unknown command "rewards" for "query"`)

I am running archwayd version 0.0.5, is it the correct one?

Reply:
@lufa23 the RPC node points to the Titus testnet which uses the latest build from the archway **"main"** repo. The 0.0.5 release only works for the Constantine network.

Reply:
so, it looks like the[ 0.0.5 release](https://github.com/archway-network/archway/tree/8e3e5beabc0d531a0a03462c1b7594aa98b98b70) has only the gastracker module

Launching **git status** i get:
`HEAD detached at 547c9a5,` which corresponds to the `547c9a524588cc3b0e9fd0668a86904a749d8dbc` commit that @drewstaylor mentioned to use for the Constantine fork


i am not sure why, as launching `archwayd q --help` i can see the following modules:


```
account
auth
authz
bank
block
distribution
evidence 
fee
grant  
gastracker 
gov   
ibc 
ibc-transfer  
mint   
params
slashing 
staking 
tendermint-validator-set
txs
upgrade
wasm
```


In any case, has the reward module been deployed to Constantine?
Because IF it wasn't AND we discourage users to use Titus, then i guess we should deploy the rewards modules to Constantine before merging these docs

Reply:
@lufa23 This docs site update refers to the protocol upgrade that's currently live on Titus (and will be enabled on Constantine soon). Additionally, the protocol team will be fixing the release numbers soon so that the protocol upgrade is correctly tagged `v0.0.6`. 

Hope that clarifies.

Reply:
great, thanks
I have added a warning about it. When the updates will be deployed to Constantine too, i will remove the warning